model/task_queue_service.go

```diff
@@ -60,16 +60,38 @@ func (s *taskDispatchService) FindNextTask(distroID string, spec TaskSpec) (*Tas
 }
 
 func (s *taskDispatchService) RefreshFindNextTask(distroID string, spec TaskSpec) (*TaskQueueItem, error) {
+	start := time.Now()
 	distroDispatchService, err := s.ensureQueue(distroID)
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
+	grip.Debug(message.Fields{
```

Comment:
Would it make sense to make these `DebugWhen` for some value (greater than a second?) to reduce logging and narrow it down to slow things?